config/crd/bases/k8ssandra.io_k8ssandraclusters.yaml

```diff
@@ -12166,13 +12166,22 @@ spec:
                                               filters to MCAC in order to reduce the
                                               amount of extracted metrics. Not setting
                                               this field will result in the default
-                                              filters being used. Setting it to an
```

Comment:
The changes in this file and in other similar ones do not look related to the nil pointer issue at hand, do they?

Reply:
I don't think they are related, no. I think the CRs got bumped automatically during the `make test` process (which has a dependency on the `manifests` target). 

Someone must have forgotten to bump them in previous iterations, but only the description is affected so no functionality will have been harmed.